File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        hash_map = defaultdict(int)
        for i in range(len(nums)):
            if target - nums[i] in hash_map:
                return [hash_map[target-nums[i]], i]
            else:
                hash_map[nums[i]] = i
       
     
                    
                    
                    
                    
                    
                    
                    
                    

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> ans = {-1,-1};
        for(int i = 0; i < nums.size(); i ++){
            for(int j = 0; j < nums.size(); j++ ){
                if(j != i && nums[i] + nums[j] == target){
                    ans  = {i,j};
                    return ans;
                }
            }
        
        }
        return ans;
        
    }
};